config/schemas/ccr_claim_schema.json

```diff
@@ -95,40 +95,56 @@
           },
           "number_of_cases": {
             "id": "/properties/bills/items/properties/number_of_cases",
-            "type": "integer"
+            "type": "string"
```

Comment:
Is this because CCR needs strings?

Reply:
Had a long slack chat with clem about this and we have settled on using strings for any number. BigDecimals, in particular, default to strings via the BigDecimal.to_json method. There is quite alot of chatter about this stuff e.g. https://github.com/rails/rails/issues/25017 . rails used to have an override but it was removed for sensible reasons. I made integers strings too firstly for consistency (CCR can just assume strings and handled appropriately) but also was reading that the JSON datatype integer is not parsed consistently by some libraries in any event - meaning you could put a float/decimal in and might get away with it.

Reply:
👍